### DISCUSSION TITLE
x-text ignore computed property changes

### DISCUSSION THREAD

**Discussion by Akiyamka** [alpinejs/alpine#3706] [7. Bugs (archived)] [answered]
## Example:
https://jsfiddle.net/akiyamka/kcn97jqf/6/

## Description 
I expect that computed property `count` will be recalculated after `combinations` change, but this not happens

**Comment by Akiyamka** [comment, marked as answer]
Computed property from other computed property must call prev prop as function.
Was (count not works)
```js
function setupState() {
  return {
    invariants: "🦅, 🐍, 🐋",
    slots: 3,
    combinations() {
      return onChange(this.invariants, this.slots);
    },
    count() {
      return this.combinations.length
    }
  };
}
```

Fixed version:
```js
function setupState() {
  return {
    invariants: "🦅, 🐍, 🐋",
    slots: 3,
    combinations() {
      return onChange(this.invariants, this.slots);
    },
    count() {
      return this.combinations().length
    }
  };
}
```

**Comment by ekwoka** [comment]
You need to add the word `get` before the methods you want to act as properties.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get

Just a JavaScript issue :)

Not sure what `onChange` is...
